docs/code-samples/lifecycling/daml.yaml

```diff
@@ -19,6 +19,7 @@ data-dependencies:
   - .lib/daml-finance-interface-instrument-base.dar
   - .lib/daml-finance-interface-instrument-bond.dar
   - .lib/daml-finance-interface-instrument-token.dar
+  - .lib/daml-finance-interface-instrument-types.dar
```

Comment:
need to upgrade the `.template` file a well. Also, can you update it for the other projects as well?